Gemfile

```diff
@@ -11,6 +11,7 @@ gem 'sinatra-contrib'
 gem 'rake'
 gem 'rack-contrib'
 gem 'graphql'
+gem 'dotenv'
```

Comment:
we don't need dotenv in production, all configuration will be passed to docker container